OWL2/StaticAnalysis.hs

```diff
@@ -351,8 +351,9 @@ createAxioms s fl = do
 check1Prefix :: Maybe String -> String -> Bool
 check1Prefix ms s = case ms of
     Nothing -> True
-    Just iri -> iri == s
-
+    Just iRi -> iRi == s || "<"++iRi++">" == s || iRi == "<"++s++">"
+      -- todo: ensure that addition of angle brackets is not necessary
```

Comment:
Maybe this todo needs to be addressed.

Reply:
made sure that angle brackets are removed, if present, before comparing strings